cmd/mailchain/internal/http/handlers/send_message.go

```diff
@@ -253,7 +254,7 @@ func isValid(p *PostRequestBody, protocol, network string) error {
 	}
 
 	// TODO: be more general when getting key from hex
-	p.publicKey, err = secp256k1.PublicKeyFromHex(p.Message.PublicKey)
+	p.publicKey, err = secp256k1.PublicKeyFromBytes(hexutil.Decode(p.Message.PublicKey))
```

Comment:
The build is failing due to `hexutil.Decode(p.Message.PublicKey)` returning 2 values, `([]byte, error)` you will need to define the response values and check the errors above before passing them here